src/main/java/frc/robot/subsystems/elevatorRoller/factory/RealElevatorRollerConstants.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems.elevatorRoller.factory;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.filter.Debouncer;
+import frc.robot.constants.IDs;
+import frc.robot.hardware.digitalinput.supplied.SuppliedDigitalInput;
+import frc.robot.hardware.motor.talonsrx.TalonSRXMotor;
+import frc.robot.hardware.signal.supplied.SuppliedDoubleSignal;
+import frc.robot.subsystems.elevatorRoller.ElevatorRollerStuff;
+import com.revrobotics.SparkLimitSwitch;
+
+import java.util.function.BooleanSupplier;
+
+public class RealElevatorRollerConstants {
+
+	private final static SparkLimitSwitch.Type REVERSE_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
```

Comment:
If not using sparkmax why having this constant?

Reply:
forgot to remove it (in the start I thought it was sparkmax, then changed it and forgot)